frontend/packages/dev-console/src/components/pipelineruns/logs/pipelineRunLogSnippet.ts

```diff
@@ -4,26 +4,11 @@ import {
   PLRTaskRunData,
   PLRTaskRunStep,
 } from '../../../utils/pipeline-augment';
+import { CombinedErrorDetails } from './log-snippet-types';
 import { pipelineRunStatus } from '../../../utils/pipeline-filter-reducer';
+import { taskRunSnippetMessage } from './log-snippet-utils';
```

Comment:
You created a generic `log-snippet-utils`... perhaps we just move this content to that file instead of having a PLR one and a "generic" one that only contains TaskRun functionality. Just merge the two and we don't need to worry about it.

Reply:
well, I think it is elaborate enough. `taskRunSnippetMessage` which is fundamental message block for both the taskSnippets. So, if you look close, the final output of both the LogSnippet a PipelineRun/ TaskRun is to return a message as per the relevant taskRun container/pod or a static one.